data_structures/src/chain.rs

```diff
@@ -1312,6 +1380,19 @@ impl TransactionsPool {
             retain
         });
     }
+
+    /// Get transaction by hash
+    // FIXME(918): only vtt and drt are supported, commits and reveals are not indexed by hash
+    pub fn get(&self, hash: &Hash) -> Option<Transaction> {
+        self.vt_transactions
+            .get(hash)
+            .map(|wvt| Transaction::ValueTransfer(wvt.1.clone()))
```

Comment:
wvt?

Reply:
WeightedValueTransfer, I will change that